README.md

```diff
@@ -659,7 +660,7 @@ Hazelcast C++ client supports the following data structures and features:
 * Queue
 * Set
 * List
-* MultiMap
+* multi_map
```

Comment:
Did not we change the names of others as well? Map ,Queue?

Reply:
Yes, they are changed. I updated them now.